runtime/dancebox/src/lib.rs

```diff
@@ -199,6 +202,8 @@ pub mod opaque {
     pub type Block = generic::Block<Header, UncheckedExtrinsic>;
     /// Opaque block identifier type.
     pub type BlockId = generic::BlockId<Block>;
+    /// Opaque block hash type.
+    pub type Hash = <BlakeTwo256 as HashT>::Output;
```

Comment:
I needed this type to use it inside `service.rs`, and I added it directly on dancebox. I can add it to flashbox as well but I won't use it from there, and at the end it would be the same type tho.

Reply:
if you are not going to use it that is fine, dont add it